src/hooks/use-add-site.ts

```diff
@@ -65,6 +65,7 @@ export function useAddSite() {
 					clearImportState( newSite.id );
 				}
 				await startServer( newSite.id );
+				updateSite( { ...newSite, isAddingSite: false } );
```

Comment:
@fluiddot , thanks for reviewing it. I agree with your concern, and at first I thought it would require a big refactor or moving the state management into another function, but then I found an actual good solution by passing a function as callback. Let me know what you think.
https://github.com/Automattic/studio/pull/456/commits/84c286fee9258ff0f696dde05506882d6c25a4b4

Reply:
Oh, good point. Passing a callback looks like a solid approach to me. Thanks for addressing this 🙇 !